osu.Game/Screens/Edit/Verify/VerifyScreen.cs

```diff
@@ -118,10 +122,11 @@ protected override void LoadComplete()
 
             private void refresh()
             {
-                var issues = generalVerifier.Run(Beatmap);
+                var workingBeatmap = beatmapManager.GetWorkingBeatmap(beatmap.BeatmapInfo);
+                var issues = generalVerifier.Run(beatmap.PlayableBeatmap, workingBeatmap);
 
                 if (rulesetVerifier != null)
-                    issues = issues.Concat(rulesetVerifier.Run(Beatmap));
+                    issues = issues.Concat(rulesetVerifier.Run(beatmap.PlayableBeatmap, workingBeatmap));
```

Comment:
Any reason for passing `EditorBeatmap.PlayableBeatmap` through instead of `EditorBeatmap`? Internally it forwards everything, but it's probably better this way to keep things consistent?